src/server/detail/snapshot_storage.h

```diff
@@ -0,0 +1,110 @@
+// Copyright 2023, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+
+#pragma once
+
+#include <filesystem>
+#include <string>
+#include <string_view>
+#include <utility>
+
+#include "io/io.h"
+#include "server/common.h"
+#include "util/cloud/aws.h"
+#include "util/fibers/fiberqueue_threadpool.h"
+#include "util/uring/uring_file.h"
+
+namespace dfly {
+namespace detail {
+
+namespace fs = std::filesystem;
```

Comment:
you can move this alias to cc.

Reply:
sure - its used by `void SubstituteFilenameTsPlaceholder(fs::path* filename, std::string_view replacement);` signature - will update that to use use `std`